docs/supported_docker_environment/windows.md

```diff
@@ -23,6 +25,17 @@ effort.
 
 * WCOW is currently not supported, since Testcontainers uses auxiliary Linux containers for certain tasks and Docker for Windows does not support hybrid engine mode at the time of writing.
 
+## Windows Subsystem for Linux
+
+Testcontainers supports communicating with Docker for Windows within the Windows Subsystem for Linux *([**WSL**](https://docs.microsoft.com/en-us/windows/wsl/about))*.
+The following additional configurations steps are required:
+
++ Expose the Docker for windows daemon on the `2375` tcp port without **TLS**.
++ Set the `DOCKER_HOST` environment variable as `tcp://localhost:2375`.
++ Modify the `/ect/wsl.conf` file to mount the windows drivers on `/` instead of on `/mnt/`.
```

Comment:
```suggestion
+ Modify the `/ect/wsl.conf` file to mount the windows filesystem on `/` instead of on `/mnt/`.
```

Reply:
I disagree with this one.
Both Docker for Windows and Nick's post refer to these as **Drives** _(I used drivers by mistake, will fix it)_.

Reply:
Shall we go with `drives` then? I don't see too much of a clarity difference, as long as the accidental typo of `drivers` is fixed.

Reply:
@rnorth It should have been fixed, as well as all the other comments in https://github.com/testcontainers/testcontainers-java/pull/1361/commits/42529b260e2c7bb098651547efdba2e9345506d9 & in https://github.com/testcontainers/testcontainers-java/pull/1361/commits/37ff88bd40e9ae7d6a5efdf53993f24492084607

I would be happy to fix any other comment you have :)
_(however, I deleted the fork due inactivity)_